### PR TITLE
G2 2023 v7 issue#14201

### PR DESCRIPTION
**Pull request by a21jeaha**
an explanation of what the file does can be found inside the created micro service.
the added code to dugga.js will be removed after the code is approved.

### CODE REVIEW

File: DuggaSys/microservices/shared_microservices/getUid_ms.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ function getUid(){
         if(isset($_SESSION['uid'])){
             $userid=$_SESSION['uid'];
         }else{
-            $userid="1";
+            $userid="guests";
```
Comment (b21kurar) [inline DuggaSys/microservices/shared_microservices/getUid_ms.php:14]: I am a bit unsure of this change, some services require `1`. Since this issue is not related to this microservice, should this really be altered? Maybe it is needed for the MOTD microservice?

**Review by b21kurar — CHANGES_REQUESTED** [review]
Just a small caveat about the `getUid_ms.php`

File: DuggaSys/microservices/shared_microservices/getUid_ms.php (hunk 1 above)
Comment (a21jeaha) [reply DuggaSys/microservices/shared_microservices/getUid_ms.php:14]: the problem is that there already exist a user with userid 1, and his a super user

image

**Review by b21kurar — APPROVED** [review]
all good!
